canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -1166,9 +1202,15 @@ export default class PropertiesController {
 	*   filterHidden: true - filter out values from controls having state "hidden"
 	*   filterDisabled: true - filter out values from controls having state "disabled"
 	*   filterHiddenControls: true - filter out values from controls having type "hidden"
+	*   getActiveTabControls: true - return all properties under selected tab/category
 	*/
 	getPropertyValues(options) {
-		const propertyValues = this.propertiesStore.getPropertyValues();
+		// All property values
+		let propertyValues = this.propertiesStore.getPropertyValues();
+
+		if (options && options.getActiveTabControls) {
```

Comment:
I would probably rename this to `activeTabValues` instead.  getActiveTabControls makes me think this is a method.